src/main/java/core/basesyntax/figure/AbstractFigure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.figure;
+
+public class AbstractFigure {
+    protected String color;
+
+    public AbstractFigure(String color) {
+        this.color = color;
+    }
+
+}
```

Comment:
The class name `AbstractFigure` suggests that it should be an abstract class, but it's currently a regular class. If the intention is to provide a common base for all figures, consider making this an abstract class or renaming it to follow the convention if it's not meant to define abstract behavior.